src/test/java/org/nebula/example/HeaderPageObject.java

```diff
@@ -0,0 +1,74 @@
+package org.nebula.example;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HeaderPageObject extends BasePageObject {
+    private final BaseTestingUtil util;
+    private FeaturesPageObject featurePO;
+    @FindBy(css = "ul#menu-main-menu>li:nth-child(1)>a")
+    private WebElement featuresItem;
+
+    @FindBy(css = "ul#menu-main-menu>li:nth-child(2)>a")
+    private WebElement benefitsItem;
+
+    @FindBy(css = "ul#menu-main-menu>li:nth-child(3)>a")
+    private WebElement aboutItem;
+
+    @FindBy(css = "ul#menu-main-menu>li:nth-child(4)>a")
+    private WebElement insightsItem;
```

Comment:
and if they change order of items, all these selectors will break. I would use `a[href='#benefits']`